serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -9,22 +9,22 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
-ARG version=12.2.2-cudnn8-devel-ubuntu22.04
+ARG version=12.4.1-cudnn-devel-ubuntu22.04
```

Comment:
you may want to use the one without cudnn and install cudnn8 manually (CUDNN 9 is installed by default

Reply:
Updated with manual install of cudnn8